sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/analysis/Analyzer.scala

```diff
@@ -1428,8 +1428,36 @@ class Analyzer(
       // SPARK-25942: Resolves aggregate expressions with `AppendColumns`'s children, instead of
       // `AppendColumns`, because `AppendColumns`'s serializer might produce conflict attribute
       // names leading to ambiguous references exception.
-      case a @ Aggregate(groupingExprs, aggExprs, appendColumns: AppendColumns) =>
-        a.mapExpressions(resolveExpressionTopDown(_, appendColumns))
+      case a: Aggregate =>
+        val planForResolve = a.child match {
```

Comment:
nit: move the original comment `// SPARK-25942: ....` above this line.

Reply:
> nit: move the original comment `// [SPARK-25942](https://issues.apache.org/jira/browse/SPARK-25942): ....` above this line.

Done

Reply:
The comment is lost?

Reply:
> The comment is lost?

Oh make a mistake, add back